Docs/docs/gettingStarted.md

```diff
@@ -12,8 +12,8 @@
 
 ## **Prerequisites**
 
-* P
-* VS Code
+* P (version >= 2.0.15)
```

Comment:
Maybe have commands they can run instead of telling the version similar to on the P webpages. 
https://p-org.github.io/P/getstarted/install/